sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/PreDefinedTagData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary> Tag details. </summary>
+    public partial class PreDefinedTagData
```

Comment:
this should inherit from subresource?  If Id is a resource id like /subscription/00000/tagNames/myName

Reply:
Done